### PR TITLE
Fix interval types by downcasing type for string matching

### PR DESCRIPTION
**Pull request by ryantking**
Found a small bug where Presto (tested using the [official Presto image](https://hub.docker.com/r/prestosql/presto)) returns interval type names in all capitals (e.g. `INTERVAL YEAR TO MONTH` versus `bigint`). Since this package compares the type names to lowercase strings, this PR will downcase type names to avoid case differences causing incorrect type name matching.

### CODE REVIEW

**Review by agrawaldevesh — APPROVED** [review]

**Comment by aweisberg** [comment]
@agrawaldevesh I added Presto committers to this repository so you are technically able to merge this PR if you are comfortable with it.

**Comment by agrawaldevesh** [comment]
Test In Prod :-)

**Pull request merged**
